### PR TITLE
Add member info/photo for Kevin Liu

### PR DESCRIPTION
**Pull request by kevnlliu**
…as a developer for Patriots and Paws

### CODE REVIEW

File: website/src/app/views/our-team/members.js [hunk 1]
```diff
@@ -183,6 +184,12 @@ const members = {
         linkedinURL: 'https://www.linkedin.com/in/michellelin1/',
         imageURL: michelleblin,
       },
+      {
```
Comment (claudeisakeeb) [inline website/src/app/views/our-team/members.js:187]: Members should be shown in ascending alphabetical order. Could you move your picture to the right spot?

**Review by claudeisakeeb — CHANGES_REQUESTED** [review]
Changes look good, just one reordering issue and this PR should be good to go

**Review by alanchangxyz — CHANGES_REQUESTED** [review]
yeah what claude said + pls fix merge conflicts + your pr title shouldn't be this long LOL (you can move the full thing to the description and make the title something short like `Add member info/photo for Kevin Liu`)

**Review by claudeisakeeb — APPROVED** [review]
Looks great! LGTM

**Pull request merged**